TrackerAlignment/src/AlignmentDerivatives.cc

```diff
@@ -6,90 +6,99 @@
 
```

Comment:
I'm going to suggest again moving this to gen, especially if we aren't using it for now

Reply:
I'll definitely need to figure this out for the next PR.